### ISSUE TITLE
NTR: Liver biopsy: dihydropyrimidinase activity low (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9258]
New term request Liver biopsy: dihydropyrimidinase activity low (elthat_111121061901): Hepatic biopsy

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000083
- label: Reduced hepatic dihydropyrimidinase activity
- parents: [Abnormal liver enzyme activity or concentration (HP:0034685)](https://hpo.jax.org/app/browse/term/HP:0034685)
- pmid: PMID:9323563
- def: Activity of dihydropyrimidinase in the liver below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9258
- synonyms:
